### PR TITLE
V12.1.0

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Thank you for your contribution to Go-AutoRest! We will triage and review it as soon as we can.

As part of submitting, please make sure you can make the following assertions:
 - [ ] I've tested my changes, adding unit tests if applicable.
 - [ ] I've added Apache 2.0 Headers to the top of any new source files.
 - [ ] I'm submitting this PR to the `dev` branch, except in the case of urgent bug fixes warranting their own release.
 - [ ] If I'm targeting `master`, I've updated [CHANGELOG.md](https://github.com/Azure/go-autorest/blob/master/CHANGELOG.md) to address the changes I'm making.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Fixes https://github.com/Azure/go-autorest/issues/394

**Comment by jhendrixMSFT** [comment]
@feiyushi FYI

File: autorest/azure/environments.go [hunk 1]
```diff
@@ -135,6 +137,7 @@ var (
 			Datalake:            NotAvailable,
 			Batch:               "https://batch.core.usgovcloudapi.net/",
 			OperationalInsights: "https://api.loganalytics.us",
+			Storage:             "https://storage.azure.com/",
```
Comment (vladbarosan) [inline autorest/azure/environments.go]: so these are the same in all env ?

Comment (jhendrixMSFT) [reply autorest/azure/environments.go]: Correct, see https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities#azure-storage-blobs-and-queues

**Pull request merged**